src/PlatformPeg.ts

```diff
@@ -18,10 +18,10 @@ limitations under the License.
 import BasePlatform from "./BasePlatform";
 
 /*
- * Holds the global instance of the `Platform` to use across the codebase
- * (singleton). Looking for an `Platform`? Just look for the `PlatformPeg` on
- * the peg board. "Peg" is the literal meaning of something you hang something
- * on. So you'll find a `Platform` hanging on the `PlatformPeg`.
+ * Holds the current instance of the `Platform` to use across the codebase
```

Comment:
err, sorry: PlatformPeg and EventIndexPeg do appear to be singletons. It's just the MatrixClientPeg that isn't.

Reply:
I think the description still holds up. Anything to do here?

Reply:
I'm torn between "let's just send it" and "we should fix this", honestly. I'd prefer it if we were accurate to the usage of the classes so we don't accidentally misuse the peg or accidentally think there's a bug, but wording that is challenging.

I guess my short term solution would be to undo the "like a singleton" change for the two pegs mentioned.

Reply:
I've removed the singleton language from all three. I think it's worth having but I don't think any of them match your take of a singleton so we can move forward ⏩

 - `PlatformPeg`: Has a `set` to change the instance
 - `EventIndexPeg`: This one seems close. Has an `unset` to remove the instance
 - `MatrixClientPeg`: New instance every time `createClient` is called. Also has an `unset`
 
It's all `getInstance` type pattern which a singleton would have though.